Config-template.toml

```diff
@@ -1,33 +1,3 @@
-<<<<<<< HEAD
```

Comment:
Please compare this with what is on the main branch. The config template should contain all (relevant) options and comments explaining them. Also confirm that the template actually works by renaming it to `Config.toml` and starting the backend. (If I remember then some of the TOML headings need to be present for the backend to start at all).